docs/builtins/General/base64-decode.md

```diff
@@ -1,31 +1,32 @@
 ## base64-decode
-Use `base64-decode` to decode a `string` from unpadded base64 encoding.
+
+Use `base64-decode` to convert a previously-encoded `string` from unpadded base64 encoding to a string.
```

Comment:
@rsoeldner can we confirm that this is URL-safe or standard alphabet? Might as well add in the useful context while we have time.